src/test/resources/goal-output/freecol-0.11.6/net.sf.freecol.common.model.Player_goal.json

```diff
@@ -8437,7 +8437,7 @@
     "parameters": [],
     "returnTag": {
       "comment": "True if this object has been disposed.",
-      "condition": "",
+      "condition": "true ? result=target.disposed",
```

Comment:
If I'm not wrong, disposed is a private field in a superclass, then is not visible.